lib/cog/commands/help/command_formatter.ex

```diff
@@ -1,74 +1,34 @@
 defmodule Cog.Commands.Help.CommandFormatter do
-  alias Cog.Models.{BundleVersion, CommandOption, CommandOptionType, CommandVersion}
+  alias Cog.Models.{CommandOption, CommandOptionType, CommandVersion}
 
-  def format(%CommandVersion{bundle_version: %BundleVersion{config_file: %{"cog_bundle_version" => version}}, documentation: documentation})
-      when version < 4 do
-    """
-    ```
-    #{String.trim_trailing(to_string(documentation))}
-    ```
-    """
-  end
+  def render_synopsis(%CommandVersion{options: options, arguments: arguments} = command_version) do
+    name = CommandVersion.full_name(command_version)
+    options = render_synopsis_options(options)
 
-  def format(%CommandVersion{} = command_version) do
-    body = command_version
-    |> render_sections
+    [name, options, arguments]
     |> Enum.reject(&is_nil/1)
-    |> Enum.map(&String.trim_trailing/1)
-    |> Enum.join("\n\n")
-
-    """
-    ```
-    #{body}
-    ```
-    """
-  end
-
-  defp render_sections(%CommandVersion{} = command_version) do
-    [render_name(command_version),
-     render_description(command_version),
-     render_synopsis(command_version),
-     render_required_options(command_version),
-     render_options(command_version),
-     render_subcommands(command_version),
-     render_examples(command_version),
-     render_notes(command_version),
-     render_author(command_version),
-     render_homepage(command_version)]
+    |> Enum.join(" ")
   end
 
-  defp render_name(%CommandVersion{description: description} = command_version) do
-    name = [CommandVersion.full_name(command_version), description]
-    |> Enum.reject(&is_nil/1)
-    |> Enum.join(" - ")
-
-    """
-    NAME
-      #{name}
-    """
+  def render_required_options(%CommandVersion{options: options}) do
+    options
+    |> Enum.filter(&(&1.required))
+    |> Enum.map(&render_option/1)
   end
 
-  defp render_description(%CommandVersion{long_description: nil}),
-    do: nil
-  defp render_description(%CommandVersion{long_description: long_description}) do
-    """
-    DESCRIPTION
-    #{indent(long_description)}
-    """
+  def render_options(%CommandVersion{options: options}) do
+    options
+    |> Enum.reject(&(&1.required))
+    |> Enum.map(&render_option/1)
   end
 
-  defp render_synopsis(%CommandVersion{options: options, arguments: arguments} = command_version) do
-    name = CommandVersion.full_name(command_version)
-    options = render_synopsis_options(options)
-
-    synopsis = [name, options, arguments]
-    |> Enum.reject(&is_nil/1)
-    |> Enum.join(" ")
-
-    """
-    SYNOPSIS
-      #{synopsis}
-    """
+  def render_subcommands(%CommandVersion{subcommands: nil}),
```

Comment:
Now that I think about it, I think this only comes up during testing when the `subcommands` key isn't provided. I'll fix that.